src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -256,6 +257,12 @@ private boolean launchNode(SchedulerDriver driver, Offer offer,
     //executorName is to distinguish different types of nodes
     //taskType is the type of task in mesos to launch on the node
     //taskName is a name chosen to identify the task in mesos and mesos-dns (if used)
```

Comment:
I don't think we need this map as it complicates the code. We can compute the name on the fly using getNextTaskType as before. What is the purpose of this change?